website/static/js/pages/institution-page.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+
+ko.punches.enableAll();
+
+var Projects = require('js/myProjects.js').MyProjects;
+var LinkObject = require('js/myProjects.js').LinkObject;
```

Comment:
I thiiink webpack is smart enough to handle the duplicate import, but just to be sure, better to do

```js
var _myProjects = require('js/myProjects');
var Projects = _myProjects.MyProjects;
//...
```